packages/moleculer-db-adapter-mongoose/test/unit/index.spec.js

```diff
@@ -166,6 +166,31 @@ describe("Test MongooseStoreAdapter", () => {
 			});
 		});
 
+		it("call stringToObjectID", () => {
+			mongoose.Types.ObjectId.isValid = jest.fn(() => true);
+			mongoose.Schema.Types.ObjectId= jest.fn();
```

Comment:
Can you please lint the code and fix the warnings?

Reply:
The only eslint errors i see relate to the other adapters.